dist/paper-core.d.ts

```diff
@@ -0,0 +1 @@
+import './paper';
```

Comment:
https://eslint.org/docs/rules/eol-last

Reply:
Indeed, I didn't notice that yet but the rest of the files seems to follow this rule (even if not reported by the linter task).
I corrected that in the next commit.